pkg/grpcserver/grpcserver.go

```diff
@@ -153,6 +153,11 @@ func (s *GrpcServer) IsRunning() bool {
 	return s.running
 }
 
+// Listener returns the listener used for this gRPC server
```

Comment:
Can you add this receiver to https://github.com/libopenstorage/grpc-framework/blob/master/pkg/grpc/server/grpcserver.go so that we don't loose track of it?

Reply:
https://github.com/libopenstorage/grpc-framework/pull/28